CHANGELOG.md

```diff
@@ -1,11 +1,28 @@
 # CHANGELOG
 
 -----
+## `v4.1.0-beta`
+
+| api                           | version             | note     |
+|:------------------------------|:--------------------|:---------|
+| arm/network                   | 2016-09-01          | updated  |
```

Comment:
Are there any reasons for updating version of all the services if no changes detected? Just curious about teams initial reasoning.